src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/SessionTemplateRepository.kt

```diff
@@ -51,6 +51,21 @@ interface SessionTemplateRepository : JpaRepository<SessionTemplate, Long> {
     @Param("visitsToDate") visitsToDate: LocalDate?,
   ): Int
 
+  @Query(
+    "select count(*) from visit v " +
+      " JOIN session_template st ON st.reference = v.session_template_reference " +
+      " WHERE st.reference = :reference" +
+      " AND v.visit_start > :visitsFromDate" +
+      " AND (cast(:visitsToDate as date) is null OR v.visit_start < :visitsToDate)" +
```

Comment:
<= here?

Reply:
Yeha the visitToDate is exclusive so, buy looking at the code we have used this convention intermittently

Reply:
looking at this we are gaining one more min thats all